### PR TITLE
Update context string to follow #195

### PR DESCRIPTION
**Pull request by nyaxt**

### CODE REVIEW

**Comment by irori** [comment]
I think we should use a draft-specific string as noted in the spec, such as `"HTTP Exchange 1 b1"`.

File: go/signedexchange/signer.go [hunk 1]
```diff
@@ -15,6 +15,10 @@ import (
 	"github.com/WICG/webpackage/go/signedexchange/cbor"
 )
 
+// The "context string" in Step 7.2 of
+// https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#signature-validity
+const ContextString = "HTTP Exchange 1"
```
Comment (hajimehoshi) [inline go/signedexchange/signer.go]: Following the convention of Go, comments should start with `ContextString is...`.

Comment (hajimehoshi) [reply go/signedexchange/signer.go]: Do we need to export this const? If not, this should be `contextString`

Comment (nyaxt) [reply go/signedexchange/signer.go]: Done

**Pull request merged**